gaeguli/pipeline.c

```diff
@@ -56,6 +56,8 @@ struct _GaeguliPipeline
   gboolean show_overlay;
 
   guint stop_pipeline_event_id;
+
+  GstCaps *source_device_caps;
```

Comment:
You shouldn't need these caps, see below.

Reply:
Yes, you are right.